lib/application-processing/schema.graphql

```diff
@@ -30,3 +30,9 @@ type UpdateApplicationProcessingResult {
 type CompleteApplicationResult {
   ok: Boolean!
 }
+
+type ApplicationFileAttachments {
+  documentUrls: [String!]
```

Comment:
```suggestion
  documentUrls: [String!]!
```

Reply:
Left as nullable for now. Filtering by a null array field seems to be pretty challenging with Prisma (I can't figure out how to do it at least) so its causing a lot of errors to make this required.